src/Sylius/Bundle/ShippingBundle/Resources/config/app/state_machine.yml

```diff
@@ -22,3 +22,4 @@ winzou_state_machine:
             cancel:
                 from: [ready]
                 to: cancelled
+        callbacks:
```

Comment:
Its working?

Reply:
no :( this line should be deleted, callback logic is moved to :`src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_shipment.yml`